docs/ops/arithmetic/Add_1.md

```diff
@@ -12,29 +12,29 @@
 
   * **Description**: specifies rules used for auto-broadcasting of input tensors.
   * **Range of values**:
-    * *none* - no auto-broadcasting is allowed, all input shapes should match
-    * *numpy* - numpy broadcasting rules, aligned with ONNX Broadcasting. Description is available in <a href="https://github.com/onnx/onnx/blob/master/docs/Broadcasting.md">ONNX docs</a>.
+    * *none* - no auto-broadcasting is allowed, all input shapes must match
+    * *numpy* - numpy broadcasting rules, description is available in <a href="https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/movement/Broadcast_1.md">Broadcast op</a>.
   * **Type**: string
   * **Default value**: "numpy"
   * **Required**: *no*
 
 **Inputs**
 
-* **1**: A tensor of type T. Required.
-* **2**: A tensor of type T. Required.
+* **1**: A tensor of type T and arbitrary shape and rank. Required.
+* **2**: A tensor of type T and arbitrary shape and rank. Required.
 
 **Outputs**
 
-* **1**: The result of element-wise addition operation. A tensor of type T.
+* **1**: The result of element-wise addition operation. A tensor of type T, if `auto_broadcast` is `numpy` shape and rank are determined by multidirectional broadcasting, otherwise they are the same as the inputs. When both inputs are of rank = 0 result is a scalar.
 
 **Types**
 
 * *T*: any numeric type.
 
 **Detailed description**
```

Comment:
_Detailed description_ section should be moved after _Short Description_.

Reply:
Corrected.